O/OpenCC/build_tarballs.jl

```diff
@@ -0,0 +1,28 @@
+using BinaryBuilder
+
+name = "libopencc"
```

Comment:
This name ***must*** match the name of the directory, so you have to either change this name or rename the directory.  I'd suggest doing the former
```suggestion
name = "OpenCC"
```